app/Policies/DocuSignEnvelopePolicy.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Policies;
+
+use App\Models\DocuSignEnvelope;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class DocuSignEnvelopePolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     */
+    public function viewAny(User $user): bool
+    {
+        return true;
+    }
+
+    /**
+     * Determine whether the user can view the model.
+     */
+    public function view(User $user, DocuSignEnvelope $docuSignEnvelope): bool
+    {
+        return true;
```

Comment:
Should this restrict to themselves and `can-access-nova`?

Reply:
Well, the primary contact and officers need to be able to view it as well. But i can restrict to those groups.